digitalstrategie/assets/scss/components/_measures.scss

```diff
@@ -10,3 +10,14 @@
 .meta-information__column__head {
     font-weight: bold;
 }
+
+@media screen and (max-width: 37.4375rem) {
+    .meta-information__row {
+        flex-direction: column;
+        margin: 0;
+    }
+
+    .meta-information__column {
+        margin-bottom: ($spacer * 2);
+    }
```

Comment:
this spacing looks a bit bigger then on the designs tested on chrome, but I also struggled to get the actual amount from zeplin because of the elements being grouped, what do you think?